Documentation/OTHER-FUNCTIONS.md

```diff
@@ -0,0 +1,63 @@
+# Other Fastly_Cdn Functions
+
+## Contents
+
+* [Purging](#purging)
+* [VCL Design Exceptions](#vcl-design-exceptions)
+
+## Purging
+
+Fastly caches objects for a certain period of time according to their TTL.
+After that the object will not be requested from the web server or Magento
+again. Until the TTL expires Fastly will deliver the cached object no matter
+what will change within Magento or the webserver's file system. To force Fastly
+to cleanup its' cache and to retrieve the information again from the backend
+you can trigger a purge request right from Magento.
+
+In the Magento2 backend go to System -> Cache Management. If you have enabled
+the Fastly_Cdn module in the configuration you will see three new buttons
+"Additional Cache Management" section.
+
+### Purge by content type
+
+You can purge objects in Fastly's Cache based on its content type by just
+clicking "Clean Fastly CDN Cache by content type". This will allow you for
+example to remove the CSS files of all store views in Fastly if they have been
+modified without the need to invalidate any other object which will save a lot
+of resources on high frequented stores.
+
+### Purge by store
+
+You can purge objects in Fastly's Cache based on the store by just clicking
+"Clean Fastly CDN Cache by store". This will only remove content that is
+generated from Magento. Images, CSS files or JavaScripts will not be purged.
+
+### Purge a URI
```

Comment:
The header says `URI`, but the text says `URL`. Seeing as the application uses `URL`, change the heading to `URL`?